itn-installer.sh

```diff
@@ -102,7 +103,4 @@ echo
 echo "To check the logs"
 echo "tail -F /var/log/rusk.{log,err}"
```

Comment:
Since 2dc38c7e64c6dee999b56a6d4222473d482595d8:

```suggestion
echo "tail -F /var/log/rusk.log"
```